trigger/task_test.go

```diff
@@ -163,6 +165,7 @@ func (s *taskSuite) SetupTest() {
 		FinishTime:          startTime.Add(20 * time.Minute),
 		RevisionOrderNumber: 1,
 		Requester:           evergreen.RepotrackerVersionRequester,
+		Details:             apimodels.TaskEndDetail{Description: evergreen.TaskDescriptionStranded},
```

Comment:
Adding this to SetupTest means that every existing test is going to have stranded details, which might break other tests (or at least not be relevant). Instead you can just update the task document as part of your test.